executors/keptn/pkg/keptn/testwith/helmrelease.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v2beta1
```

Comment:
What's all the `testwith` code doing this deep? Also, would help to have an understanding of what exactly this is doing.

Reply:
This was mostly for reference - its more docs like that testwith. Let me move it into the `executors/keptn` dir instead as a documentation dir.